cs-utilities/src/test/java/io/cloudslang/content/utilities/FindTextInPdfTest.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * (c) Copyright 2017 Hewlett-Packard Enterprise Development Company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+
+package io.cloudslang.content.utilities;
+
+import io.cloudslang.content.utilities.actions.FindTextInPdf;
+import org.junit.Test;
+
+import java.io.File;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Map;
+
+import static io.cloudslang.content.constants.OutputNames.RETURN_CODE;
+import static io.cloudslang.content.constants.OutputNames.RETURN_RESULT;
+import static io.cloudslang.content.constants.ReturnCodes.FAILURE;
+import static io.cloudslang.content.constants.ReturnCodes.SUCCESS;
+import static junit.framework.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by marisca on 7/11/2017.
+ */
+public class FindTextInPdfTest {
+    private final FindTextInPdf findTextInPdf = new FindTextInPdf();
+
+    @Test
+    public void samplePdfSuccessfulTest() {
+        try {
```

Comment:
why try { } catch ? ... that catch will hide fails 😆  ... add it to the method signature